scripts/play.ts

```diff
@@ -0,0 +1,380 @@
+import mri from "mri";
+import chalk from "chalk";
+import * as fs from "node:fs/promises";
+import * as fsSync from "node:fs";
+import * as path from "node:path";
+import * as crypto from "node:crypto";
+
+import { exec } from "./utils/commandUtils";
+
+const PLAYGROUNDS_ROOT = path.join(process.cwd(), "playgrounds");
```

Comment:
> ❓ Would [this](https://stackoverflow.com/a/50052194) more modern approach work?

It's slightly different in that `process.cwd()` is the working directory, while `import.meta.url` is the file's URL. The former should decouple the script's location from its internal logic, while `import.meta.url` forces the script to know its location.

In this case, `import.meta.url` will work since we know the script will always live in `scripts`. It will let us replace `path.join` with something more platform-agnostic.

> ❓ Would [this](https://stackoverflow.com/a/50052194) more modern approach work?
> 
> 💡 It seems "root" refers to both the `playgrounds` parent directory and the directory of a bootstrapped project. Would renaming `PLAYGROUNDS_ROOT` to `PLAYGROUNDS_DIR` help better differentiate the 2 concepts?

Yes, good call. I was getting confused myself.